manifests/kustomize/env/cert-manager/platform-agnostic-multi-user-k8s-native/kustomization.yaml

```diff
@@ -28,3 +28,41 @@ patches:
     target:
       kind: MutatingWebhookConfiguration
       name: pipelineversions.pipelines.kubeflow.org
+  - patch: |
+      - op: add
+        path: "/rules/-"
+        value:
+          verbs:
+            - create
+            - get
+            - list
+            - watch
+            - update
+            - patch
+            - delete
+          apiGroups:
+            - pipelines.kubeflow.org
+          resources:
+            - pipelines
+            - pipelines/status
+      - op: add
+        path: "/rules/-"
+        value:
+          verbs:
+            - create
+            - get
+            - list
+            - watch
+            - update
+            - patch
+            - delete
+          apiGroups:
+            - pipelines.kubeflow.org
+          resources:
+            - pipelineversions
+            - pipelineversions/status
+    target:
+      group: rbac.authorization.k8s.io
+      version: v1
+      kind: Role
```

Comment:
This needs to be a ClusterRole for multi-user since the pipelines can be in any namespace.